Candidate: CVE-2015-5159
PublicDate: 2018-10-30 18:29:00 UTC
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5159
 https://bugzilla.redhat.com/show_bug.cgi?id=1245200
 https://github.com/latchset/kdcproxy/commit/f274aa6787cb8b3ec1cc12c440a56665b7231882
Description:
 python-kdcproxy before 0.3.2 allows remote attackers to cause a denial of
 service via a large POST request.
Ubuntu-Description:
Notes:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]


Patches_python-kdcproxy:
upstream_python-kdcproxy: released (0.3.2)
precise/esm_python-kdcproxy: DNE
trusty_python-kdcproxy: DNE
trusty/esm_python-kdcproxy: DNE
xenial_python-kdcproxy: not-affected (0.3.2-1)
esm-apps/xenial_python-kdcproxy: not-affected (0.3.2-1)
bionic_python-kdcproxy: not-affected (0.3.2-1)
esm-apps/bionic_python-kdcproxy: not-affected (0.3.2-1)
cosmic_python-kdcproxy: not-affected (0.3.2-1)
devel_python-kdcproxy: not-affected (0.3.2-1)
